main.cpp

```diff
@@ -57,6 +57,7 @@ void usage(std::ostream &os) {
         "-r [ --rate ] arg      Refresh rate override\n"
         "-m [ --mirror ]        mirror outputs using the lowest common resolution\n"
         "-o [ --order ] arg     order of outputs, repeat as needed\n"
+        "-c [ --copy ] arg      arg in format dst:src, copy from output src to output dst, repeat as needed\n"
```

Comment:
Perhaps "copy resolution, refresh and position using format dst:src, repeat as needed"